### PR TITLE
BaSP-MR-23: Test for methods get/getbyid/post for members

### PR DESCRIPTION
**Pull request by NicolasKler**
I did 14 test between get, get by id and post, using a seed with 3 mock members

### CODE REVIEW

File: src/tests/members.test.js [hunk 1]
```diff
@@ -1,3 +1,105 @@
...
+    expect(response.error).toBeFalsy();
+    expect(response.body.data.length).toBe(3);
+  });
+  test('should return status 404', async () => {
+    const response = await request(app).post('/api/Member/invalid').send();
+    expect(response.status).toBe(404);
+    expect(response.error).toBeTruthy();
+  });
+  test('should create an employee with status 201', async () => {
+    const response = await request(app).post('/api/Members').send(mockMember);
```
Comment (fernandofernandezuy) [inline src/tests/members.test.js:38]: i think you should have consistence in the route, you have somes routes with '/Members' and others with '/members'

**Review by leo-balonchard — APPROVED** [review]
Looks good

File: src/tests/members.test.js (hunk 1 above)
Comment (NicolasKler) [reply src/tests/members.test.js]: Names changed

**Review by fernandofernandezuy — APPROVED** [review]
Looks good to me!, its ready to merge.

**Pull request merged**